lib/rbac/authorizer.rb

```diff
@@ -57,8 +51,7 @@ def user_role_allows?(user, options = {})
       end
     end
 
-    def user_role_allows_any?(user, options = {})
-      return false if user.miq_user_role.nil?
```

Comment:
are we guaranteed to have a role here and above?  Were they useless guard clauses?

Reply:
moved the guard clause up. will roll that back to simplify this PR